cognite/neat/_session/_template.py

```diff
@@ -206,6 +211,8 @@ def extension(self, io: Any, output: str | Path | None = None) -> IssueList:
             io: The input spreadsheet.
             output: The output spreadsheet. If None, the output will be the same
                 as the input with `_extension` added to the name.
+            dummy_property: The dummy property to use as placeholder for user-defined properties
+                            for each user-defined concept.
```

Comment:
Is it worth mentioning filtering as well? Something like
```suggestion
            dummy_property: The dummy property to use as placeholder for user-defined properties
                            for each user-defined concept. When converting a data model, it is recommended to have at least
                            one property for each concept. This ensures that you follow that recommendation.
```